requirements.txt

```diff
@@ -3,5 +3,10 @@ Django==3.1.5
 django-cors-headers==3.6.0
 django-webpack-loader==1.0.0
 djangorestframework==3.12.2
+joblib==1.0.1
+more-itertools==8.8.0
+nltk==3.6.2
 pytz==2020.5
+regex==2021.4.4
 sqlparse==0.4.1
+tqdm==4.61.1
```

Comment:
We now don't need these dependencies in this PR, but since we'll be turning around quickly and using them, I think it's fine to include them here.